Algorithms/Other/GeoLocation.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Algorithms.Other
+{
+    public static class GeoLocation
+    {
+        private const double EarthRadius = 6371.01d;
+
+        /// <summary>
+        /// Calculate distance from 2 point of latitude, longitude from https://www.movable-type.co.uk/scripts/latlong.html.
+        /// </summary>
+        /// <param name="lat1">Latitude of point A.</param>
+        /// <param name="lng1">Longitude of point A.</param>
+        /// <param name="lat2">Latitude of point B.</param>
+        /// <param name="lng2">Longitude of point B.</param>
+        /// <returns>Distance between point A to point B.</returns>
+        public static decimal CalculateDistanceFromLatLng(double lat1, double lng1, double lat2, double lng2)
+        {
+            var result = 0m;
+
+            try
+            {
+                if (Math.Abs(lat1 - lat2) == 0d && Math.Abs(lng1 - lng2) == 0)
```

Comment:
Codacy found an issue: [Do not check floating point equality with exact values, use a range instead.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6396283)